test/stripe/api_resource_test.rb

```diff
@@ -182,11 +195,37 @@ class NestedTestAPIResource < Stripe::APIResource
         Stripe::Charge.list(:count => nil, :offset => 5, :sad => false)
 
         stub_request(:post, "#{Stripe.api_base}/v1/charges").
-          with(body: {  'amount' => '50', 'currency' => 'usd' }).
+          with(body: { 'amount' => '50', 'currency' => 'usd' }).
           to_return(body: JSON.generate({ :count => 1, :data => [API_FIXTURES.fetch(:charge)] }))
         Stripe::Charge.create(:amount => 50, :currency => 'usd', :card => { :number => nil })
       end
 
+      should "putting idempotency_key correctly as an opt should not trigger a warning" do
```

Comment:
So the naming of these test cases should be readable from the "should".

"should putting idempotency_key correctly ..." doesn't make sense. Maybe something like "should trigger a warning ...".